### ISSUE TITLE
Only one battery reported

### ISSUE THREAD

**Issue by loustak** [Gerschtli/dwm-status#11]
Hello,

Since I updated to master from and older version, dwm-status now only report one of my two batteries.
The result of `acpi` is the following:
```
Battery 0: Discharging, 83%, 02:37:07 remaining
Battery 1: Unknown, 78%
```

**Comment by Gerschtli** [comment]
Recently I have replaced the logic which was responsible for gathering battery information with the following crate: https://github.com/svartalf/rust-battery

Unfortunately I don't have a laptop with two batteries so I can not test it, but maybe you could report your issue there, as they use a much more sophisticated solution than I had in the first place..

Maybe you could debug the return value in `src/wrapper/battery.rs` to see which crate is causing the problem..

**Comment by loustak** [comment]
Thanks for pointing this out, I will investigate.

**Comment by Gerschtli** [comment]
Should be fixed via #12 and will be released in the next days.

**Comment by Gerschtli** [comment]
Fix is published in 1.7.1
